test/enrichedFactories.js

```diff
@@ -65,7 +65,7 @@ export const makeGroup = data => {
     status: 'active',
     notificationTypes: [
       'weekly_summary',
-      'daily_pickup_notification',
+      'daily_activity_notification',
```

Comment:
Don't change yet, is part of API (gets passed to backend)

Reply:
Fixed!